src/utils/columnUtils.ts

```diff
@@ -189,3 +189,16 @@ export function getColumnScrollPosition<R, SR>(columns: readonly CalculatedColum
 
   return 0;
 }
+
+/**
+ * By default, the following navigation keys are enabled while an editor is open, under specific conditions:
+ * - Tab:
+ *   - The editor must be an <input>, a <textarea>, or a <select> element.
+ *   - The editor element must be the only immmediate child of the editor container.
```

Comment:
Would it break if input has a container element like a label?
div.rdg-editor-container > label > input